src/sentry/migrations/0779_remove_groups_from_group_inbox.py

```diff
@@ -0,0 +1,61 @@
+# Generated by Django 5.1.1 on 2024-10-24 19:51
+
+from django.apps.registry import Apps
+from django.db import migrations
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+
+from sentry.new_migrations.migrations import CheckedMigration
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBarApprox
+
+BATCH_SIZE = 100
+
+
+class GroupStatus:
+    RESOLVED = 1
+    IGNORED = 2
+
+
+def remove_groups_from_group_inbox(apps: Apps, schema_editor: BaseDatabaseSchemaEditor) -> None:
+    GroupInbox = apps.get_model("sentry", "GroupInbox")
+    Group = apps.get_model("sentry", "Group")
+
+    group_ids_to_delete = []
+    for group_inbox in RangeQuerySetWrapperWithProgressBarApprox(GroupInbox.objects.all()):
```

Comment:
It might be helpful to have `.select_related("group")` here so you can just do `group_inbox.group.status` and avoid a second query.